test/helpers/browser.js

```diff
@@ -7,6 +7,7 @@ var vm = require('vm');
 
 function createContext() {
   var context = {
+    self: this,
```

Comment:
I see. In that case `context.self = context;` is correct, see https://developer.mozilla.org/en-US/docs/Web/API/Window/self

>  By using `self`, you can refer to the global scope in a way that will work not only in a window context (self will resolve to `window.self`) but also in a worker context (self will then resolve to `WorkerGlobalScope.self`).